### PR TITLE
Rework Docker images according to new specs

### PR DESCRIPTION
**Pull request by pzduniak**
forgot to switch the branch before starting work :/

### CODE REVIEW

File: packaging/linux/docker/build.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ build_dir="$1"
 here="$(dirname "$BASH_SOURCE")"
 client_dir="$(git -C "$here" rev-parse --show-toplevel)"
 source_commit="$(git -C "$here" log -1 --pretty=format:%h)"
-tag="$(cat "$build_dir/VERSION" | tr "+" "-")"
+tag="$(echo $1 | tr "+" "-")"
```
Comment (heronhaye) [inline packaging/linux/docker/build.sh]: maybe check this value before to make sure its not empty

File: packaging/linux/docker/standard/entrypoint.sh [hunk 2]
```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env bash
+set -e
```
Comment (heronhaye) [inline packaging/linux/docker/standard/entrypoint.sh]: euo pipefail?

File: packaging/linux/docker/slim/entrypoint.sh [hunk 3]
```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -e
```
Comment (heronhaye) [inline packaging/linux/docker/slim/entrypoint.sh]: euo pipefail?

File: packaging/linux/docker/slim/entrypoint.sh [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+# If a command was passed and KEYBASE_SERVICE is not set, simply run the command
+if [ "$#" -gt 1 ] -a [[ -z "${KEYBASE_SERVICE}" ]]; then
+    exec "$@"
+    exit 0
+fi
+
+# If we're called with any args, don't pollute stdout and stderr and write the
+# logs into /var/log/keybase
+if [ "$#" -gt 1 ]; then
+    bash -c "keybase ${KEYBASE_SERVICE_ARGS:-"-debug"} service &> /var/log/keybase/service.log" &
```
Comment (heronhaye) [inline packaging/linux/docker/slim/entrypoint.sh]: why does it shell out?
also if we redirect logs like that I don't think log sends will work - is that ok?

File: packaging/linux/docker/build.sh (hunk 1 above)
Comment (pzduniak) [reply packaging/linux/docker/build.sh]: `set -u` already checks it

File: packaging/linux/docker/standard/entrypoint.sh (hunk 2 above)
Comment (pzduniak) [reply packaging/linux/docker/standard/entrypoint.sh]: k ill do full set and i might revert it if it breaks

File: packaging/linux/docker/slim/entrypoint.sh (hunk 4 above)
Comment (pzduniak) [reply packaging/linux/docker/slim/entrypoint.sh]: i _think_ it should work (you'll get 2 log files) and i can change it in the future after we do some basic tests

i'll drop the shellout, it should probably work as-is without bash -c

**Pull request merged**